client/src/components/reportThankyou.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
```

Comment:
Can we use reactjs-popup module?

Reply:
I hope so, the slack group seems to think it is ok.